test/e2e/postinstall/machinesets/infra_test.go

```diff
@@ -22,12 +25,19 @@ import (
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	hivev1aws "github.com/openshift/hive/pkg/apis/hive/v1/aws"
 	hivev1azure "github.com/openshift/hive/pkg/apis/hive/v1/azure"
+	hivev1gcp "github.com/openshift/hive/pkg/apis/hive/v1/gcp"
 	"github.com/openshift/hive/test/e2e/common"
 )
 
+const (
+	workerPoolName       = "worker"
```

Comment:
Either
`workerPoolName` and `infraPoolName`
or
`workerMachinePoolName` and `infraMachinePoolName`.